tests/unity/CMakeLists.txt

```diff
@@ -145,9 +150,9 @@ function(cmock_linker_wrap_trick header_file_path)
   cmake_parse_arguments(CMOCK "" "" "EXCLUDE" ${ARGN})
   set(flist_file "${header_file_path}.flist")
 
-  if(DEFINED CMOCK_EXCLUDE)
-    string(JOIN "--exclude;" exclude_arg ${CMOCK_EXCLUDE})
-    set(exclude_arg "--exclude" ${exclude_arg})
+  if(DEFINED CMOCK_FN_EXCLUDE)
```

Comment:
From where does `CMOCK_FN_EXCLUDE` originate ?

The function variables starting with `CMOCK_<argument>` are created by the `cmake_parse_arguments` concatenated with `CMOCK` + `_` + arg name, such as `EXCLUDE`.

But there is not `FN_EXCLUDE` argument.
Ref: https://github.com/nrfconnect/sdk-nrf/blob/1812edc3ef875fb1c28a786b197b7bf2d860cb21/tests/unity/CMakeLists.txt#L150

Reply:
I see, should have fixed it now